projects/ngx-vest-forms/src/lib/directives/validation-options.directive.ts

```diff
@@ -0,0 +1,25 @@
+import { Directive, input } from '@angular/core';
+
+/**
+ * Define validation options for a form, form model group or form model
+ */
+@Directive({
+  selector: '[validationOptions]',
```

Comment:
I'd also make the selector more specific, it can only used on elements that have an [ngModel] or ngModelGroup directive right?

Reply:
No need for this anymore as I deleted the directive